buckwheat/tests/test_files/test.py

```diff
@@ -2,6 +2,7 @@
 
 from tree_sitter import Language, Parser
 
+# No parsers need to be declared here
```

Comment:
do we have single and multi-line comments for each language?

Reply:
https://twitter.com/gvanrossum/status/112670605505077248

Reply:
Add multi-line comments for all languages except Shell. Out of the box Shell doesn't support multi-line comments.

Reply:
what is the type of this multiline string/comment in python?
```
"""
bla
"""
```